tests/system/test_query_retry.py

```diff
@@ -0,0 +1,46 @@
+import contextlib
```

Comment:
license headers :-)

Reply:
Fixed, here and in unit test. Sorry.

Reply:
Needs to be the Google header in this repo. https://opensource.google/docs/releasing/preparing/#Apache-header